### PR TITLE
Bug fixing #24

### PR DESCRIPTION
**Pull request by KassandraA**
All bugs are fixed.

### CODE REVIEW

File: source/js/view.js [hunk 1]
```diff
@@ -104,18 +104,19 @@ function updateSymbolsList() {
...
+            let _k = k
+            
             symbolLi.appendChild(symbolImg);
             symbolImg.setAttribute('src', 'images/' + symbolsImageArray[availableSymbols[i]] + '.png');
             symbolImg.setAttribute('alt', availableSymbols[i]);
-            symbolImg.addEventListener('click', function() {
-                activeSymbol[k].removeChild(activeSymbol[k].firstChild);
-                activeSymbol[k].appendChild(symbolImg.cloneNode(true));
-                playerSymbolUList[k].querySelector('li').removeAttribute('class');
+            symbolImg.addEventListener('click', function() {
```
Comment (aannenko) [inline source/js/view.js]: Redundant spaces.

File: source/js/view.js [hunk 2]
```diff
@@ -444,8 +421,8 @@ function doResize() {
             playFieldBody.style.top = (playField.offsetHeight - (plFieldTransformer.offsetHeight * scale)) / 2 + 'px';
         }
 
-        if (transformer[i] === settingsTransformer) {
-            settingsTransformer.style.height = (window.innerHeight * 0.8) / scale + 'px';
+        if (transformer[i] === settingsTransformer) {
```
Comment (aannenko) [inline source/js/view.js]: Redundant spaces

File: source/js/view.js [hunk 3]
```diff
@@ -444,8 +421,8 @@ function doResize() {
             playFieldBody.style.top = (playField.offsetHeight - (plFieldTransformer.offsetHeight * scale)) / 2 + 'px';
         }
 
-        if (transformer[i] === settingsTransformer) {
-            settingsTransformer.style.height = (window.innerHeight * 0.8) / scale + 'px';
+        if (transformer[i] === settingsTransformer) {   
+            settingsTransformer.style.height =  wrapper[i].offsetHeight / scale + 'px';
```
Comment (aannenko) [inline source/js/view.js]: Redundant space.

**Review by aannenko — CHANGES_REQUESTED** [review]

**Comment by KassandraA** [comment]
Changes are done.

**Comment by KassandraA** [comment]
Changed favicon.

**Comment by aannenko** [comment]
Nice job!

**Pull request merged**